config/initializers/rack-attack.rb

```diff
@@ -69,7 +60,7 @@ def user_id
   # Throttle requests for device_flow_authorize
   #
   # Key: "rack::attack:#{Time.now.to_i/:period}:oauth/device_flow_authorize:#{req.user_id}"
-  throttle("oauth/device_flow_authorize", :limit => 1, :period => 5.seconds) do |req|
+  throttle("oauth/device_flow_authorize", :limit => 1000, :period => 5.seconds) do |req|
```

Comment:
Propose increasing the period to say 15 minutes, since it is unlikely that the class size would exceed that when registering, and reduces our risk to ddos